### DISCUSSION TITLE
Partnership

### DISCUSSION THREAD

**Discussion by carolandreazza** [naverz/zepeto-studio-kor#485] [4. General]
Hi!!
I work at a marketing agency and we would love to partner with Zepeto to put our agency in there (clothes and interacting world) and also call our customers.
Is there such a possibility?
How do I do?
I already sent a partnership email, but no one responded

**Comment by Zepeto-TechEvangelist** [comment]
Hello! Thank you for your interest in Zepeto 😄
Regarding the partnership, please contact us by email below.

dl_partnerships@naverz-corp.com

**Reply by carolandreazza** [reply]
Hello! Email sent! Thanks

**Comment by carolandreazza** [comment]
Hi! Could you let me know if my partnership email has already been read? was it accepted or not?

**Reply by Zepeto-TechEvangelist** [reply]
Sorry I'm not in charge of partnership, so I can't answer it
If you have sent me an email, it may take a long time to consider it
Please wait :)

**Reply by carolandreazza** [reply]
Sorry but I need to ask again, can you check if my proposal email has been read? if there was any return?
It is very important for us that this partnership be carried out. We want to bring our customers to Zepeto too.
 Is there another way I can talk? even if it's something paid, please

**Reply by Zepeto-TechEvangelist** [reply]
Hello, What's the name of your company or team?
I will inform the department in charge.

**Reply by carolandreazza** [reply]
Hi, it's Jump. I don't remember if i just put that or the whole name Jump Aceleradora Digital.
Thanks.

**Comment by Zepeto-TechEvangelist** [comment]
The person in charge confirmed that there was no e-mail under the name of jump, jump Aceleradora Digital.
Please email dl_partnerships@naverz-corp.com again.

**Reply by carolandreazza** [reply]
Hi. Strange because I have it here as sent, but I sent it again with the name of your company Jump Projetos Disruptivos (we recently changed the name). Awaiting return. Thanks
